de.marlam.qv.yml

```diff
@@ -0,0 +1,166 @@
+app-id: de.marlam.qv
+runtime: org.kde.Platform
+runtime-version: '6.4'
+sdk: org.kde.Sdk
+command: qv
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # OpenGL rendering
+  - --device=dri
+  # Read any scientific data file (mostly stored outside the xdg hierarchy)
+  # and save a visualization alongside it
+  - --filesystem=host
+cleanup:
+  # '/bin/[a-p]*' does not work, so list explicitly
+  - '/bin/M*'
```

Comment:
you could add the cleanup `/bin` in the specific module that create them. It would be simpler

Reply:
Done, thanks.